src/components/Dropdown/Dropdown.tsx

```diff
@@ -423,7 +427,9 @@ export const Dropdown: FC<DropdownProps> = React.memo(
           {getReference()}
           <ConditionalWrapper
             condition={portal}
-            wrapper={(children) => <FloatingPortal>{children}</FloatingPortal>}
+            wrapper={(children) => (
+              <FloatingPortal preserveTabOrder>{children}</FloatingPortal>
```

Comment:
this will help portal component stay in current tab order? If so could you add it to all places using FloatingPortal?

Reply:
seems like this is already default true based on doc https://floating-ui.com/docs/floatingportal, is this doing anything?

Reply:
I cannot say if its doing anyhting because I could not verify this in storybook. This seem to be a viable fix so i added it.  Was not aware that it is true by default.